### ISSUE TITLE
Reconsider the process.env.SKIP_SASS_BINARY_DOWNLOAD_FOR_CI

### ISSUE THREAD

**Issue by saper** [sass/node-sass#1453]
I've always had some doubts about this variable but during the v3.5.0 release process (#1451) it became obvious to me: why have a separate variable only to stop the download, if another (`SASS_FORCE_BUILD` or `-f` flag) is used to force the build anyway?

So, we have those effects:

SKIP_SASS_BINARY_DOWNLOAD_FOR_CI=1 and SASS_FORCE_BUILD=0 : no download, the build will be started due to empty `vendor` directory if the binary hasn't been planted there otherwise (manual install)

SKIP_SASS_BINARY_DOWNLOAD_FOR_CI=0 and SASS_FORCE_BUILD=1: download occurs, but the build proceeds anyway, overwriting the downloaded file (if successful).

Maybe we should use only SASS_FORCE_BUILD and drop the unofficial SKIP_SASS_BINARY_DOWNLOAD_FOR_CI ?

**Comment by xzyfer** [comment]
There's an interesting use case for `SKIP_SASS_BINARY_DOWNLOAD_FOR_CI` documented in https://github.com/sass/node-sass/issues/1037#issuecomment-122701332.

>Maybe we should use only SASS_FORCE_BUILD and drop the unofficial SKIP_SASS_BINARY_DOWNLOAD_FOR_CI ?

I'd be curious to see if this has any side effects. I predict slightly slower builds which isn't a bug deal.

**Comment by orjan** [comment]
Not sure if this is related to this issue but sitting behind a flaky corporate proxy it would be great to be able to specify an alternative download url, similar to https://github.com/Medium/phantomjs#downloading-from-a-custom-url e.g:
```
phantomjs_cdnurl=https://bitbucket.org/ariya/phantomjs/downloads
```

**Comment by xzyfer** [comment]
@orjan this is possible and documented in the readme - https://github.com/sass/node-sass#binary-configuration-parameters

**Comment by orjan** [comment]
@xzyfer thank you very much and sorry for interrupting the flow of this issue! I can delete the comment if you want to?

**Comment by xzyfer** [comment]
That would be great :)

**Comment by xzyfer** [comment]
Closed by #1734
